docs/framework/install/dotnet-35-windows-10.md

```diff
@@ -20,6 +19,10 @@ You may see the following configuration dialog if you try to run an app that req
 
 ![.NET Framework Installation Dialog](./media/dotnet-framework-installation-dialog.jpg)
 
+### Why am I getting this pop-up?
+
+The .NET Framework is a development framework created by Microsoft and used by many companies to develop their apps. Different versions of the .NET Framework already come pre-installed in your Windows machine. When you try to run an app that was developed using version 3.5 of the .NET Framework and you don't have it installed on your machine, you are prompted to do so.
```

Comment:
I think this paragraph could use some work. Since this is a user topic, calling something a development framework is unclear and also raises the question of why, as a user, you need a development framework on your machine. I think I'd rephrase this as something like: "The .NET Framework provides an environment for running applications. Applications that run in the .NET Framework target a specific version. If you see this pop-up, you are trying to run an application that requires the .NET Framework Version 3.5, but that version is not installed on your system."

Reply:
I'll rework this. I think it's also important to call out that is developed by Microsoft. Some people think it's a virus.